apps/docs/pages/guides/integrations/snaplet.mdx

```diff
@@ -0,0 +1,151 @@
+
+</div>
+
+### Things you'll need before you begin: [#prerequisite]
+
+1. **A production Supabase project’s connection string:** This can be found in Supabase via **Organization > All Projects > Your Project > Project Settings > Connection Pooling > Connection String**. We’ll refer to this in the guide as your “source database” - where your data comes from.
+
+![Supabase connection string](/docs/img/guides/integrations/snaplet/connection_string.webp)
+
+2. A development Supabase project’s connection string: Same steps as above, but a different project/environment. We’ll refer to this in the guide as your “target database” - where we’re restoring the snapshot of the source database to. If you haven’t got this set up yet, we’ll take you through the process of setting this up.
+
+> **Highly Recommended:** Create a read-only role for your connection string. For more info on how to create a read-only role across all schemas, you can check out the [Snaplet docs](https://docs.snaplet.dev/guides/postgresql#create-a-read-only-role).
+
+### Step 1: Connect your source database (Production Database) [#connect-source-db]
+
+The first thing you’ll want to do is navigate to [https://www.snaplet.dev/](https://www.snaplet.dev/) and sign up for a new account (it’s free). Once you have successfully signed up for a new account, you’ll begin the onboarding process...
+
+![Onboarding start](/docs/img/guides/integrations/snaplet/onboarding_start.webp)
+
+On the “Connect database” step, click on "Connect Supabase" to connect your supabase account to snaplet. Proceed through all the steps of authorizing your Supabase organization then selecting Supabase project and giving Snaplet the password associated to your project.
+
+![Validating credentials](/docs/img/guides/integrations/snaplet/connect_to_supabase.webp)
+
+### Step 2: Create a snapshot [#create-snapshot]
+
+Once that is done, continue to the next steps:
+
+- Subsetting (skipping will create a snapshot with all your data).
+- Transforming (skipping will leave all your table columns untouched).
+
+Once you have gone through those steps, a new snapshot process will start and you will now be on the “capture step.” Wait for it to finish - this is the snapshot you will restore into your **target database.**
+
+### Step 3: Create a target database on Supabase (Development Database) [#create-target-db]
+
+Your target database is where you want Snaplet to restore the captured snapshot of your production project. This would most likely be either your staging or development Supabase project.
+
+If you don’t already have a development database setup on Supabase, you’ll need to create a new database by setting up a new project on Supabase. To create a new project you’ll have to do the following:
+
+1. Go to [app.supabase.io](https://app.supabase.io/)
+
+2. Go to your **organization > All projects**
+
+3. In the top left click on your current project name and new project:
```

Comment:
```suggestion
3. In the menu bar, open the project dropdown and click on 'New project`:
```

Avoid using directional descriptors as much as possible, so the docs stay robust if UI shifts around